modules/gitops-release-notes-1-3-2.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3-2_{context}"]
+= Release notes for {gitops-title} 1.3.2
+
+{gitops-title} 1.3.2 is now available on {product-title} 4.6, 4.7, 4.8, and 4.9.
+
+[id="new-features-1-3-2_{context}"]
+== New features
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.2:
+
+* Upgraded Argo CD to version *2.1.8*
+
+* Upgraded Dex to version *2.30.0*
+
+[id="fixed-issues-1-3-2_{context}"]
+== Fixed issues
+
+The following issues have been resolved in the current release:
+
+* Previously, the {gitops-title} did not appear in the `OperatorHub` when you filtered by "Disconnected" under the *Infrastructure Features*, since the Operator did not have the related annotation set in its CSV. With this update, the {gitops-title} is annotated with "Disconnected Cluster" as an *Infrastructure Feature* link:https://issues.redhat.com/browse/GITOPS-1539[GITOPS-1539]
+
+* When using an "Namespace-scoped" argocd, for example, argocd that is not scoped to *All Namepsaces* in a cluster, the {gitops-title} dynamically maintains a list of managed namespaces. These namespaces include the `argocd.argoproj.io/managed-by` label. This list of namespaces is stored in a cache in argocd *Settings > Clusters > "in-cluster" > NAMESPACES*. Before this update, if you deleted one of these namespaces, the Operator ignored that, and the namespace remained in the list. This behavior broke the *CONNECTION STATE* in that cluster configuration, and all sync attempts resulted in errors. For example:
```

Comment:
- a `Namespace-scoped` Argo CD instance, 
- example, an Argo CD instance that is...
- argocd - Argo CD names of products need to be mentioned as is with the appropriate case, space etc. Please check all the instances in the PR.
- the {gitops-title}/{gitops-title} - An article is not required in this case.
- argocd *Settings > Clusters > "in-cluster" > NAMESPACES*./*Argo CD -> Settings -> Clusters -> "in-cluster" -> NAMESPACES*. Please go through the OCP doc contributing guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference